docs/how-to/distributed-training.txt

```diff
@@ -83,30 +83,42 @@ S3. In this example, the S3 bucket name is configured via a field
           s3.download_file(s3_bucket, data_file, filepath)
       return download_directory
 
+.. _dtrain-scheduling:
 
 Scheduling Behavior
 -------------------
 
-To use distributed training, ``slots_per_trial`` must be set to a multiple
-of the number of GPUs per machine. For example, if you have a cluster of machines with
-8 GPUs each, you should set ``slots_per_trial`` to a multiple of 8, such as 8, 16, or 24.
-This ensures that the full network and interconnect bandwidth are available to
-multi-GPU workloads and results in better performance.
+The Determined master takes care of scheduling distributed training jobs
+automatically, ensuring that all of the compute resources required for a job are
+available before the job itself is launched. Users should be aware of the
+following details about scheduler behavior when using distributed training:
+
+- If ``slots_per_trial`` is smaller than the number of slots on a
+  single agent, Determined will consider scheduling multiple distributed training
+  jobs on a single agent. This is designed to improve utilization and to allow
+  multiple small training jobs to run on a single agent. For example, an agent
+  with 8 GPUs could be assigned two 4-GPU jobs, or four 2-GPU jobs.
+- Otherwise, if ``slots_per_trial`` is greater than the number of slots on a single agent,
```

Comment:
If we're going to phrase it this week, the previous "If ..." should be "smaller than or equal to", not "smaller than".